src/column.js

```diff
@@ -57,6 +57,7 @@ var Column = function(column) {
   this.field = column.field || this.id;
   this.css = column.css || this.id;
   this.escape = isUndefined(escape) ? true : !!escape;
+  this.width = column.width;
```

Comment:
I think a size attribute would be better (I have plans to use height attribute on columns to allow "lazy" rendering).